components/project-controller/development/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../base
+- https://github.com/konflux-ci/project-controller/config/default?ref=9937d7d5caf9625823889a8ba6865bcbd4343ad8
+
+images:
+- name: quay.io/redhat-appstudio/project-controller
+  newName: quay.io/redhat-appstudio/project-controller
```

Comment:
Is this the right image name on quay, trying to pull it and podman says it does not exist

Reply:
Yeah it's the correct one. I created it few days back (https://quay.io/repository/redhat-appstudio/project-controller).

Reply:
I see the repository but no image in there, is this a permission issue, you see images in that repo?

Reply:
I just pushed the image, can you check now?